Bottom: c413a690bbeb1c98df3b7f45ac645ab0941259e5
Top:    1d5d55d2e75f272d21df1282ea53f938bcb4fd07
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-05-17 14:43:10 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 1c755f7..af7ef91 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2459,7 +2459,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
